internal/certification/scripts/configure.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-source $HOME/.profile
+# shellcheck disable=SC1090,SC1091
```

Comment:
I set it to disable because it was a non-existent file when shellcheck is running.
https://github.com/koalaman/shellcheck/wiki/SC1090
https://github.com/koalaman/shellcheck/wiki/SC1091